com.unity.render-pipelines.core/Runtime/Documentation.cs

```diff
@@ -87,5 +90,16 @@ public static string GetHelpURL<TEnum>(TEnum mask = default)
 
             return helpURLAttribute == null ? string.Empty : $"{helpURLAttribute.URL}#{mask}";
         }
+
+        /// <summary>
+        /// Obtains the help URL from a type.
+        /// </summary>
+        /// <param name="type">The type decorated with the HelpURL attribute.</param>
+        /// <returns>The full URL or null if the attribute is not present.</returns>
+        public static string TryGetHelpURL(Type type)
+        {
+            var attribute = type.GetCustomAttribute<HelpURLAttribute>(false);
+            return attribute?.URL;
+        }
```

Comment:
Normally methods with `Try` return a `bool`.

So you either do it like this:

```suggestion
        public static bool TryGetHelpURL(Type type, out string url)
        {
            url = string.Empty;
            var attribute = type.GetCustomAttribute<HelpURLAttribute>(false);
            if (attribute == null)
               return false;
            url = attribute.URL;
            return true;
        }
```

or just rename the method to be `GetHelpURL`.

I will leave this up to you, is really a minor issue.

Reply:
Yeah, I was thinking about that as well. I figured returning `null` was similar, but seeing it, this is nicer indeed. I'll change it :)